ha-scenes.yaml

```diff
@@ -261,7 +312,6 @@ blueprint:
             - "Party vibes: Chinatown"
             - "Party vibes: Fairfax"
             - "Party vibes: Ibiza"
-            - "Party vibes: Malibu pink"
```

Comment:
This seems to be missing from https://github.com/Hypfer/hass-scene_presets/blob/master/custom_components/scene_presets/presets.json

Reply:
@nilsreiter Not sure you want me to this, I could try to combine both json sources into 1 output.

Reply:
@nilsreiter The integration allows users to add their own custom scene presets. See the documetation on that here: https://github.com/Hypfer/hass-scene_presets/blob/master/docs/Custom%20Presets.md